### ISSUE TITLE
Elevated baseline CPU load on GitLab

### ISSUE THREAD

**Issue by hannes-ucsc** [DataBiosphere/azul#5822]
<img width="1321" alt="Screenshot 2023-12-22 at 2 31 36 PM" src="https://github.com/DataBiosphere/azul/assets/5143256/e25c0f68-a2a6-4e60-b7c8-6e0819459340">

The increase coincides with the first time the "container" aka Docker image cleanup policy ran for the first time after we enabled it as part of PR #5750 for addressing issue #5728 with commit 323bc490.

Other GitLab users have this too: https://gitlab.com/gitlab-org/gitlab/-/issues/362914#note_1706255150

**Comment by hannes-ucsc** [comment]
Since the high baseline load aggravates the job timeout issue #5809, I've set 

<img width="433" alt="image" src="https://github.com/DataBiosphere/azul/assets/5143256/ae14b49c-eff6-4812-9cbe-5d07e8d76430"> 

on all instances for now.

**Comment by hannes-ucsc** [comment]
Setting the max. worker count to zero reduced the baseline CPU load to normal levels …

<img width="851" alt="image" src="https://github.com/DataBiosphere/azul/assets/5143256/1dac3c5a-6e4b-4b06-9809-320c493c09bd">

…but it effectively disables the cleanup.

**Comment by hannes-ucsc** [comment]
We may need to wait for a resolution to the upstream issue linked in the description.

**Comment by dsotirho-ucsc** [comment]
Assignee to monitor upstream issue.

**Comment by hannes-ucsc** [comment]
For #6316, I set the workers to 4 on all four instances which had negligible effect on the available disk space in all of them. However I also set two limits to zero: 

<img width="1361" alt="image" src="https://github.com/DataBiosphere/azul/assets/5143256/9b3cdc2b-a933-43a7-a07c-1ed7cd76cd4d">

There were no updates on the upstream blocker but disabling the two limits appears to alleviate the issue. The cleanup completed and the perpetual respawning of the Sidekiq jobs cannot be observed anymore. Baseline CPU is still elevated but only because clamscan is now running essentially all the time.
